example/Example.js

```diff
@@ -291,14 +292,29 @@ class Example extends Component {
       var latitude = "29.634933";
       var longitude = "-95.220255";
       var ipAddress = "103.238.105.167";
+      Countly.setLocation(null, city, latitude + "," + longitude, ipAddress);
+      Countly.setLocation(null, null, latitude + "," + longitude, ipAddress);
```

Comment:
remove the wrongly used samples

Reply:
These are not samples, these are test cases to see if the sdk is not crashing.

Reply:
Maybe you can use Template String for more readable code.

Reply:
Ok, I understand.

Reply:
Done